src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/DocumentState.cs

```diff
@@ -237,9 +237,8 @@ public virtual DocumentState WithTextLoader(Func<Task<TextAndVersion>> loader)
         private IReadOnlyList<DocumentSnapshot> GetImportsCore(DefaultProjectSnapshot project)
         {
             var projectEngine = project.GetProjectEngine();
-            var importFeature = projectEngine.ProjectFeatures.OfType<IImportProjectFeature>().FirstOrDefault();
             var projectItem = projectEngine.FileSystem.GetItem(HostDocument.FilePath);
-            var importItems = importFeature?.GetImports(projectItem);
+            var importItems = MvcImportItems.GetImportItems(projectEngine.FileSystem, projectItem);
```

Comment:
This wont work. Depending on the project engine it could be configured for 1_x 2_x etc so we need to use the imports specific to how the engine was configured.